house/views.py

```diff
@@ -42,11 +42,34 @@ def house_view(request):
 
     house = user.house
     expenses_list = Expenses.objects.filter(house_name=house)
-    context = {'house': house,
-               'house_expenses': expenses_list}
+    context = {'house': house, 'house_expenses': expenses_list}
     return render(request, MINE_MINE_PAGE_ROUTE, context)
 
 
+def add_house(request):
```

Comment:
You can delete this function, the new name is house_create

Reply:
fixed